docs/billing.md

```diff
@@ -23,7 +23,7 @@ curl -sSiX POST https://prism.ultraviolet.rs/createplan -H "Content-Type: applic
         "currency":<currency>
     },
     "public-plan":<public-plan>,
-    "org-id":<org-id>
+    "proj-id":<proj-id>
```

Comment:
this is not correct, but also just remove curl commands and replace with ui instead